### ISSUE TITLE
GameGuru MAX - AI Management Under Environment Effects and Edit Behaviors Button Bug?

### ISSUE THREAD

**Issue by ArgentArts** [TheGameCreators/GameGuruRepo#2826]
First of all, I am very excited to see this in the editor and not just in the TabTab menu when you run a test game! However, why is AI Management under Environment Effects? Seems like a strange place to put it to me.

Also, as a side note: If you click the Edit Behaviors button here, it currently does nothing in the editor. However, if you click the rocket icon to go to test game and then TabTab, the right-side menu will be automatically in the Behavior editor by default.

**Comment by LeeBamberTGC** [comment]
@rvanner Can you decide on a better place and then assign to Preben to code it, thanks!

**Comment by rvanner** [comment]
@LeeBamberTGC I don't think this should show in the Level Editor. Isn't it only for Test Level mode?
![image](https://user-images.githubusercontent.com/1503593/166208560-7826b7d6-7ba4-4c18-bfbe-206e15b20f49.png)

**Comment by MonkeyFrogStudio** [comment]
Actually, it would be great if we could see the nav mesh and things like the enemy line of sight IN the editor and make adjustments there, rather than having to run a test game, hit TABTAB, and try to do it there.

**Comment by LeeBamberTGC** [comment]
NOTE: And should also get to the EDIT directly after creating a new BEHAVIOR:
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/ca9fa0c6-e937-4fef-a09f-688e3b213d7c)
